agent/util-scripts/pbench-move-results

```diff
@@ -247,7 +264,7 @@ while read -r dir; do
         continue
     fi
 
-    if [[ "$script_name" == "pbench-move-results" ]]; then
+    if [[ "${script_name}" == "pbench-move-results" ]]; then
```

Comment:
Why make this change without also changing line 285?  (This looks like churn.)

Reply:
Change was originally made to bring the variable name reference in line with others.  But going forward based on other comments, this will be removed entirely.